UnoCpp/UnoCpp/Logger.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Logger.h"
+
+#include <iostream>
+
+void Logger::LogMessage(const std::string& message)
+{
+    std::cout << message;
+}
+
+void Logger::LogMessage(const std::vector<std::string>& lines)
+{
+    std::string fullMsg{};
+    for (const std::string& msg : lines)
+    {
+        fullMsg += msg;
+        fullMsg += "\n";
+    }
+    LogMessage(fullMsg);
```

Comment:
Sugestão: dentro do for chamar `LogMessage(msg + "\n")`. Nem precisaria de variável temporário `fullMsg`.

Reply:
Eu tinha feito isso, mas o Log chama std::cout, fiquei pensando se poderia ser ruim ficar fazendo I/O muitas vezes seguidas. Fiquei na dúvida se talvez compensava ter uma string inteira e passar tudo de uma vez, e ai deixei assim.

Reply:
eu acho que é uma boa dúvida pra tirarmos no canal, não tinha pensado nisso